spec/javascripts/admin/modules/unpublish-display-conditions.spec.js

```diff
@@ -95,4 +106,21 @@ describe('GOVUK.Modules.UnpublishDisplayConditions', function () {
     expect(body.querySelector('.js-unpublish-withdraw-form__consolidated').style.display).toEqual('block')
     expect(body.querySelector('.js-unpublish-withdraw-form__withdrawal').style.display).not.toEqual('block')
   })
+
+  it('should hide Public explanation section when Redirect to URL automatically is pre-checked', function () {
```

Comment:
These capitals seem a bit jarring as these don't seem like proper nouns. I'd imagine you can clean it up with quotes:

```suggestion
  it('should hide the public explanation section when |Redirect to URL automatically" is pre-checked', function () {
```